docker-compose.yml

```diff
@@ -1,22 +1,32 @@
 version: "3.9"
-# We can replace DB service later if needed with another one
 services:
+  mysql:
+    image: mysql:8.0
+    container_name: ays-db
+    environment:
+      MYSQL_DATABASE: ays
+      MYSQL_USER: ays
+      MYSQL_PASSWORD: ayspass
+      MYSQL_ROOT_PASSWORD: ayspass
+      MYSQL_ROOT_HOST: '%'
+      MYSQL_PORT: 3306
+    volumes:
+      - ./db:/var/lib/mysql
+    ports:
+      - 3306:3306
+
   ays-be:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: ays-be
     ports:
-      - "8080:8080"
-    environment:
-      - SPRING_PROFILES_ACTIVE=${SPRING_PROFILES_ACTIVE}
+      - 9790:9790
```

Comment:
We need to define the port as follows:
```
    ports:
       - "9790:9790"
```

When I checked, it was required to be defined within double quotation marks.